yarn.lock

```diff
@@ -5334,9 +4433,9 @@ __metadata:
   languageName: unknown
   linkType: soft
 
-"@fluentui-react-native/text@npm:*, @fluentui-react-native/text@workspace:*, @fluentui-react-native/text@workspace:packages/components/Text":
+"@fluentui-react-native/text@npm:*, @fluentui-react-native/text@workspace:*, @fluentui-react-native/text@workspace:packages/components/text":
```

Comment:
mm. I think that's because Saad changed the capitalization but Windows doesn't really care so it doesn't force the recapitalization to happen locally, and then we run into this :).

Reply:
TIL Windows doesn't care about this quirk... but like.. yarn windows I guess